<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Danielaswindow</class>
 <widget class="QMainWindow" name="Danielaswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>garderobe amigo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup/squid.png</normaloff>../../.designer/backup/squid.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-3</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 yellow, stop:1 purple);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="naskiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>ნასკი</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sharvaliCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>შარვალი</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="shortebiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>შორტები</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="iubkaCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>იუბკა</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="maisuriCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>მაისური</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="svitriCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>სვიტრი</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imReadyPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>მზად ვარ</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hoodieCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>hoodie</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bezrukavkaCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>ბეზრუკავკა</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sharfiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>შარფი</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qudiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>ქუდი</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xeltatmanebiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>ხელთათმანები</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="botasebiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>ბოტასები</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qoshebiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>ქოშები</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ketebiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>კეტები</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sandlebiCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QcheckBox{
background: qlineargradient(
    spread:pad,
    x1:0, y1:0,
    x2:1, y2:0,
    stop:0 rgba(128, 0, 128, 255),   /* Purple */
    stop:1 rgba(255, 255, 0, 255)    /* Yellow */
);}</string>
    </property>
    <property name="text">
     <string>სანდლები</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_12">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_13">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_15">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel_17">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="naskiFeriLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ფერი</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="depresiaRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>დეპრესია</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="happiRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ანტიდეპრესია</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
